rl_algorithms/recurrent/utils.py

```diff
@@ -0,0 +1,135 @@
+from typing import Any, Tuple
+
+import torch
+
+from rl_algorithms.common.helper_functions import make_one_hot
+from rl_algorithms.utils.config import ConfigDict
+
+
+def infer_leading_dims(tensor: torch.Tensor, dim: int) -> Tuple[int, int, int, Tuple]:
```

Comment:
should modify docstring
- add example
- follow google style: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
resolved in commit 3856d71.